src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -169,6 +169,7 @@ export class QuestionSubmissionFormComponent implements DoCheck {
 
   handleClick(): void {
```

Comment:
Should this method name be more specific? Like `updateIsTabExpanded`?

Reply:
Good catch. Perhaps `toggleQuestionTab`?